project/apps/questions-service/src/adapters/controllers/questions.controller.ts

```diff
@@ -1,11 +1,12 @@
 import { Controller } from '@nestjs/common';
 import { MessagePattern, Payload } from '@nestjs/microservices';
-import { QuestionsService } from './questions.service';
+import { QuestionsService } from 'src/domain/ports/questions.service';
 import {
   CreateQuestionDto,
   GetQuestionsQueryDto,
   UpdateQuestionDto,
 } from '@repo/dtos/questions';
+
 @Controller()
 export class QuestionsController {
```

Comment:
Merge #22 first, I'll rename the files from `questions.*` to `question.*` after.